docs/getting-started-with-self-hosted.md

```diff
@@ -0,0 +1,186 @@
+
+Once you've done that, you can then go to your React app where you have added your card,
+log in with your wallet, and subscribe to your new topic! The email address or other information
+you used to sign up for the Community Manager topic will also be used here.
+
+(TODO: Screenshots)
+
+## Getting Your Server Credentials
+
+Before we move on to finally creating our server, we will need one more thing from Admin Portal:
+our server credentials. Notifi refers to these as your SID and Secret, which the server will need to
+provide in order to log in and send messages.
+
+To view your SID and Secret, go to the **Account Settings** page in Admin Portal, then click
+**Show Sid/Secret** in the top-right corner. In the dialog box that appears, copy the contents
+of the Sid and Secret boxes somewhere safe - you will use these later when you create your server.
+
+:::warning
+
+These credentials are only used for authenticating your own application on your own server to run
+on your own behalf. Ensure that these credentials are stored somewhere secure, that the code using these
+credentials is being run in a secure environment, and that these credentials 
+**are NEVER sent to a user's web browser**.
+
+:::
+
+## Creating your Node JS Server
+
+Finally, we will create our server that will actually send our messages! Notifi provides a Node.js
+SDK for accessing Notifi's APIs and sending messages, which we will use for our server. (If you want to
+use a different server technology, consult the documentation for our [GraphQL API](./alert-subscribe/graphql-api)
+and our [Dataplane REST API](./alert-trigger/dataplane-api/).)
+
+1. Install [Node.js](https://nodejs.org/en) version 18 or later.
+2. Run `$ npm init` in an empty directory to create your project
+3. Run `$ npm install @notifi-network/notifi-node` to add the [Notifi Node.js SDK](https://www.npmjs.com/package/@notifi-network/notifi-node)
+   to your project.
+4. Add `"type": "module"` to your project's `package.json`, since we will be using ES6 modules and top-level `await`
+5. Create a `main.js` file with the following code:
+
+```javascript
+// TODO: Add a code example
+import {
+    NotifiClient,
+    NotifiEnvironment,
+    createNotifiService, 
+    createDataplaneClient, 
+    createGraphQLClient,
+} from '@notifi-network/notifi-node';
+
+// Enter your SID/Secret from Account Settings
+// and your Event Type ID from Alert Manager here.
+// This example assumes they are stored as environment variables
+// MY_SID, MY_SECRET, and MY_EVENT_TYPE_ID
+const MY_SID = process.env.MY_SID;
+const MY_SECRET = process.env.MY_SECRET;
+const EVENT_TYPE_ID = process.env.MY_EVENT_TYPE_ID;
+
+const gqlClient = createGraphQLClient("Production");
+const notifiService = createNotifiService(gqlClient);
+const dataplaneApiClient = createDataplaneClient("Production");
+const client = new NotifiClient(notifiService, dataplaneApiClient);
+
+const { token, expiry } = await client.logIn({
```

Comment:
do we want to give them boilerplate for using the refresh jwt?

Reply:
Unfortunately, those functions haven't been added to the Notifi SDK, so I can't. I believe Eric's long term plan with the SDK is to manage the tokens internally.